.golangci.yml

```diff
@@ -49,12 +46,17 @@ linters-settings:
 linters:
   disable:
     - errcheck
+  enable:
     - gofmt
     - goimports
```

Comment:
nit: Likely don't need these linters since we have other steps that'll check/fix them, which is why these were initially disabled.